setup.py

```diff
@@ -1,12 +1,11 @@
 from setuptools import setup
-import datashader
 
 with open('requirements.txt') as f:
     install_requires = f.read().strip().split('\n')
 
 setup(name='datashader',
-      version=datashader.__version__,
```

Comment:
Ah, good catch. Updating a version number in 3 places is unfortunate, but acceptable.